src/Dialect/ONNX/ONNXOps/Tensor/Squeeze.cpp

```diff
@@ -53,14 +53,32 @@ LogicalResult ONNXCommonSqueezeOpShapeHelper<OP_TYPE>::customComputeShape(
     // Get the shape as symbols, so that we may detect if some are dynamic.
     DimsExpr dataShape;
     createIE->getShapeAsSymbols(data, dataShape);
+
+    auto countDynamicDimensions = [&](IndexExpr dim) {
```

Comment:
nit: this lambda doesn't actually count
```suggestion
    auto isDimensionDynamic = [&](IndexExpr dim) {
```